src/data/docs/commands.md

```diff
@@ -114,7 +126,7 @@ Use `quit` (or `exit` or `:q`) to terminate UCM.
 
 Use `undo` to revert the most recent change to the codebase.
 
-## `update`
+## `update
```

Comment:
You probably didn't mean to delete the closing ` ?

Reply:
Oops, nope! Fixed.